redesign/app.html

```diff
@@ -0,0 +1,162 @@
+              I Love Productivity, Sign Me Up!
+            </button>
+          </div>
+        </form>
+
+          <div class="tc dn" id="thankyou_message">
+            <h3 class="white f2 w-100 center lh-copy">Thank you for expressing your interest!</h3>
+            <p class="dwyl-mint f3 lh-copy istok-web w-80 center">
+              We will be in touch when we're ready to launch!
+            </p>
+            <h1 class="f-xxl">🚀</h1>
+          </div>
+        <!-- Submit the Form to Google Using "AJAX" -->
+        <script data-cfasync="false" type="text/javascript" src="https://cdn.rawgit.com/dwyl/html-form-send-email-via-google-script-without-server/master/form-submission-handler.js"></script>
+        <!-- END -->
+      </div>
+    </section>
+
+    <section class="dwyl-bg-mint center pt4-ns pb5-ns">
+
+      <div class="dib-ns w-50-ns dtc v-top">
+        <div class="w-80 w-60-ns white center tr">
+          <h1 class="mt0-ns mb3 pl4-ns f-xl fw9">Let's</h1>
+          <h1 class="pl4-ns mt3 f-xl fw9">Talk</h1>
+        </div>
+        <div class="w-80-ns w-90 ml3">
+          <a href="tel:+447708796446" class="link"><h2 class="tr white"><span class="dwyl-dark-gray fw9">Tel: </span>+44 (0) 7708 796446</h2></a>
+          <a href="mailto:hello@dwyl.io" class="link"><h2 class="tr white"><span class="dwyl-dark-gray fw9">Email:</span> hello@dwyl.io</h2><a/>
+          <h2 class="tr white"><span class="dwyl-dark-gray lh-copy fw9">Address:</span> 16 Palmers Road,<br>London, E2 0SY, UK</h2>
+          <p class="tr dwyl-dark-gray">(We'll get back to you within 24 hours)</p>
+        </div>
+      </div>
+
+      <div class="dib-ns bg-white pt4 pa2 shadow-4 ml5-ns w-40-ns">
+
+          <form id="gform" method="POST" class="tc tl-ns" action="https://script.google.com/macros/s/AKfycby5ZdD8HfCaz1tMVLw26qYQKEXkP8qdfwLek3TH_RYH3l1o1-Li/exec">
+          <!-- change the form action to your script url -->
+
+          <fieldset class="bn">
+            <label for="name" class="db mb2 tl pl3 pl0-ns">Your Name</label>
+            <input id="name" name="name" class="br2 ba dwyl-b--dark-gray pa2 w-90 border-box" placeholder="Tim Berners-Lee" />
+          </fieldset>
+
+          <fieldset class="bn">
+            <label for="email" class="db mb2 tl pl3 pl0-ns">Email Address</label>
+            <input id="email" name="email" class="br2 ba dwyl-b--dark-gray pa2 w-90 border-box" type="email" value="" required placeholder="tim@cern.com" />
+          </fieldset>
+
+          <fieldset class="bn">
+            <label for="message" class="db mb2 tl pl3 pl0-ns">Message</label>
+            <textarea id="message" name="message" class="br2 ba w-90 dwyl-b--dark-gray pa2" rows="10" placeholder="What problem can we solve for you?"></textarea>
+          </fieldset>
+
+          <button class="bn dib link mt2 mb3 ph3 pv3 dwyl-bg-orange white br2 f4 shadow-4 pointer montserrat">
```

Comment:
When scrolling down this page I found that the orange here was really very jarring. I start on grey, yellow and white - all good. Then I start scrolling and get a bunch of teal which is still ok because I had hints of it in the navigation.
Then all of a sudden, orange-y button. Normally I don't complain about colours. But on this page, I think it makes a big difference - I couldn't find the style guide so I couldn't make a suggestion for a different colour that would work with the colour hierarchy that I don't know very well yet.

Reply:
The orange colour doesn't bother me too much on this page but I understand what you mean @iteles and see that it's a bit more flashy than on the main page for example. Maybe we could try with the yellow colour but I'm wondering if we want to keep the colour consistent for all pages on the "say hello" button. linked to https://github.com/dwyl/dwyl-site/issues/236#issuecomment-308147254

Reply:
I agree that the colour is a bit odd, I voiced my opinion on colours here: https://github.com/dwyl/dwyl-site/issues/236#issue-234492544. Personally I favour having a small number and feel the orange is a bit random. However, Harry outlined his reasoning and at the end of that issue it seemed the consensus was to follow him. 
 
![image](https://user-images.githubusercontent.com/16775804/30420858-d570d2c8-9932-11e7-824f-9ee969e8e437.png)

I would be happy to choose an alternative for the reasons I initially gave.

Reply:
I think my reasoning actually follows Harry's well here - in this case, **the** call to action is to sign up as a beta user, so the colour for the 'Send' button on the contact form should be the secondary button colour, not the 'call to action' colour.
This is actually why I was asking about the style guide - do we have one?

Reply:
Ah I see. No we don't have a style guide. I suppose it depends on whether you think it matters that the contact button should be consistent throughout the site...

Reply:
Why don't we open a new issue for this and augment it at a later date?

Reply:
Great, @markwilliamfirth - if you make the new issue and I'll resolve the merge conflicts and then we can get this merged in.

Reply:
@Cleop 👍